### ISSUE TITLE
Move Committees outside of the Governance doc

### ISSUE THREAD

**Issue by murillodigital** [gitops-working-group/gitops-working-group#151]
Committees are continuously changing whereas governance likely pretty much set once agreed upon, lets move committees into their own .md

**Comment by murillodigital** [comment]
Should also add a bullet as to how to volunteer for a comittee

**Comment by scottrigby** [comment]
Good idea. Do you mind making a note on this pr about that? https://github.com/gitops-working-group/gitops-working-group/pull/125?

Also cross-linking the open PR on contributing #88 @alisondy and I can connect with about how to tag-team work on these two PRs 😺
